### PR TITLE
IE, Embed, and Header

### PR DESCRIPTION
**Pull request by mwernimont**
IE SVG CSS Fix

Embed Area

Header title a little bigger on desktop

@jiwalker-usgs I'll figure out the merge conflicts

### CODE REVIEW

**Comment by mwernimont** [comment]
@jiwalker-usgs All ready to review

File: layout/templates/embedTemplate.mustache [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div id="embedSection" class="responsiveGrid">
+  {{#embeds}}
+    <h4>{{description}}</h4>
+    <div class="embedHolder">
+      <div class="embed">
+        <div class="socialMediaImage">
+          {{{blackEmbedIcon}}}
+        </div>
+      </div>
+      <iframe src="{{{url}}}"/>
```
Comment (jiwalker-usgs) [inline layout/templates/embedTemplate.mustache]: iframe should be inside input, not the other way around

**Comment by jordansread** [comment]
@mwernimont fix conflicts?

**Comment by mwernimont** [comment]
@jiwalker-usgs Got blocked by github, but should be dec for the moment

**Pull request merged**